src/transitions/scroll-transition.js

```diff
@@ -83,20 +103,25 @@ export default class ScrollTransition extends React.Component {
   getSlideStyles(index, positionValue) {
     const targetPosition = this.getSlideTargetPosition(index, positionValue);
     return {
-      position: 'absolute',
-      left: this.props.vertical ? 0 : targetPosition,
-      top: this.props.vertical ? targetPosition : 0,
+      boxSizing: 'border-box',
       display: this.props.vertical ? 'block' : 'inline-block',
-      listStyleType: 'none',
-      verticalAlign: 'top',
-      width: this.props.vertical ? '100%' : this.props.slideWidth,
       height: 'auto',
-      boxSizing: 'border-box',
-      MozBoxSizing: 'border-box',
+      left: this.props.vertical ? 0 : targetPosition,
+      listStyleType: 'none',
+      marginBottom: this.props.vertical ? this.props.cellSpacing / 2 : 'auto',
       marginLeft: this.props.vertical ? 'auto' : this.props.cellSpacing / 2,
       marginRight: this.props.vertical ? 'auto' : this.props.cellSpacing / 2,
       marginTop: this.props.vertical ? this.props.cellSpacing / 2 : 'auto',
-      marginBottom: this.props.vertical ? this.props.cellSpacing / 2 : 'auto'
+      MozBoxSizing: 'border-box',
+      position: 'absolute',
+      top: this.props.vertical ? targetPosition : 0,
+      transform:
+        this.props.animation === 'zoom' && this.props.currentSlide !== index
```

Comment:
This inline logic works fine while there's only one animation type. In the future, I expect we'll want to factor out the transform value resolution to a tested function. For now, this is appropriate! 👍

Reply:
👍 sounds good